docs/WORKFLOW_GUIDE.md

```
@@ -523,10 +524,10 @@ Profile: [esignet | mosip]
 
 | Profile | tfvars file | Cluster size | Use for |
 |---------|------------|--------------|---------|
-| `esignet` | `profiles/esignet/aws.tfvars` | 4-node K8s cluster | eSignet standalone deployment |
-| `mosip` | `profiles/mosip/aws.tfvars` | 7-node K8s cluster | Full MOSIP platform deployment |
+| `esignet-standalone` | `profiles/esignet-standalone/aws.tfvars` | eSignet standalone deployment |
```

markdownlint-cli2 (0.22.1)

[warning] Table column count
Expected: 4; Actual: 3; Too few cells, row will be missing data

(MD056, table-column-count)